docs/data/material/components/dialogs/SimpleDialog.js

```diff
@@ -31,24 +31,24 @@ function SimpleDialog(props) {
       <DialogTitle>Set backup account</DialogTitle>
       <List sx={{ pt: 0 }}>
         {emails.map((email) => (
-          <ListItem button onClick={() => handleListItemClick(email)} key={email}>
+          <ListItemButton onClick={() => handleListItemClick(email)} key={email}>
```

Comment:
This could be an opportunity to fix the DOM structure, `ul > div role button` is not valid, I would expect `ul > li > button`

Reply:
Maybe `ul > li > div[role="button]` can be better because children as `div` is not allowed inside `button`. And `role="button"` is wrong attribute value for `<li>`

Like that : 

```html
<ul>
    <li>
        <div role="button"></div>
    </li>
</ul>
```

Reply:
I pushed a commit for this.